snuba/snuba_migrations/metrics_summaries/0003_metrics_summaries_add_segment_id_duration_group_columns.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import Column, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set_name = StorageSetKey.METRICS_SUMMARIES
+
+columns: List[Column[Modifiers]] = [
+    Column("segment_id", UInt(64)),
+    Column("duration_ms", UInt(32)),
+    Column("group", UInt(64)),
```

Comment:
What is the `group`?

Reply:
`group` is the fingerprint of the span which we use in the `spans` primary key. It kind of looks like a transaction name but more specific and one group can map to multiple spans of the same sort (like the same query with different parameters).